db/model/bot.js

```diff
@@ -104,6 +104,18 @@ module.exports = function bot(seq, dataTypes) {
           foreignKey: 'botId',
           through: 'RoomTypeBots',
         });
+
+        Bot.addScope('botUI', {
+          attributes: { include: ['ui'] },
+        }, {
+          override: true,
```

Comment:
I think "override" is only necessary for the "defaultScope" ... are you sure about this here?